packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,682 @@
+  "cardinality": {
+    "min": 0,
+    "max": 1
+  }
+}' "https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}/features/{featureId}"
+```
+
+#### Response Types
+
+HTTP 200:
+[Factor Profile Feature Object](#factor-profile-feature-object)
+
+## Factor Profiles
+
+### Example
+```json
+    {
+        "id": "fpruib7klOvW4pAuK0g3",
+        "name": "Email",
+        "created": "2019-10-08T18:19:04.000Z",
+        "lastUpdated": "2019-10-08T18:19:04.000Z",
+        "settings": {},
+        "default": true,
+        "_links": {
+            "self": {
+                "href": "http://ed.okta1.com:1802/api/v1/org/factors/okta_email/profiles/fpruib7klOvW4pAuK0g3",
+                "hints": {
+                    "allow": [
+                        "GET",
+                        "PUT",
+                        "DELETE"
+                    ]
+                }
+            }
+        }
+    }
+```
+
+### Properties
+
+The Factor Profile model defines several attributes:
+
+| Property      | Type                                               | Description                                                                                                              |
+| ------------- | -------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `id`          | String                                             | Identifier of the factor profile.                                                                                        |
+| `name`        | String                                             | Name of the factor profile.                                                                                              |
+| `default`     | Boolean                                            | This is set to `true` on the profile to be used for a factor enrollment that does not have a profile associated with it. |
+| `settings`    | [Settings Object](#factor-profile-settings-object) | Settings for factor profile.                                                                                             |
+| `created`     | String (ISO-8601)                                  | Timestamp when the factor profile was created.                                                                           |
+| `lastUpdated` | String (ISO-8601)                                  | Timestamp when the factor profile was last modified.                                                                     |
+| `_links`      |                                                    | Link relations for this object                                                                                           |
+
+### Factor Profile Settings Object
+
+The factor profile settings object contains the settings for the particular factor type. Currently factor profiles can be created for two factor types: `okta_email` and `okta_password`.
```

Comment:
I wonder if it needs to be stated that while you can _create_ multiple okta password factor profiles, you can't really _consume_ any of them except for the default one currently...

Reply:
Yes, that sounds like an important point.

Reply:
Will add that here.
What about email, does the same apply? @kenwang-okta  @edjohns-okta

Reply:
I feel like we use factorName and factor types inerchangeably.
Here, we are calling `okta_email` and `okta_password` as factor types. Everywhere else in this doc it was called factor name.
cc @jakubvul

Reply:
This looks like the main difference between `factor name` and `factorType`
https://developer.okta.com/docs/reference/api/factor_admin/#factor-model

`okta_password` is a factor name. `password` is a factorType
Meaning everything defined here is a factorName https://github.com/okta/okta-core/blob/master/server/factors.interface/src/main/java/com/okta/services/factors/iface/types/OrgFactorIds.java
Everything here is a factorType https://github.com/okta/okta-core/blob/master/server/factors.interface/src/main/java/com/okta/services/factors/iface/types/FactorType.java

By that logic, it should be:

```factor profiles can be created for two factor names: `okta_email` and `okta_password`.```